Applications/CMC/test/testCMCArm26_Thelen.cpp

```diff
@@ -41,10 +41,27 @@ void testCMCArm26() {
     Storage *standard = new Storage();
     cmc.getModel().formStateStorage(temp, *standard);
 
-    Array<double> rms_tols(0.02, 2*2+2*6); // activations within 2%, angles within .6 degrees
+    Array<double> rms_tols(0, 2*2+2*6); // activations within 2%, angles within .6 degrees
     const string& muscleType = cmc.getModel().getMuscles()[0].getConcreteClassName();
     string base = "testCMCArm26 "+ muscleType;
 
+    rms_tols[0]  = 0.0002;    // r_shoulder/r_shoulder_elev/value
+    rms_tols[1]  = 0.002;     // r_shoulder/r_shoulder_elev/speed
+    rms_tols[2]  = 0.0006;    // r_elbow/r_elbow_flex/value
+    rms_tols[3]  = 0.006;     // r_elbow/r_elbow_flex/speed
+    rms_tols[4]  = 0.0000001; // TRIlong/activation
```

Comment:
How were these tolerances determined? Are we sure an RMS error of, say, 1.1e-7 is unacceptably large?